appl/gssmask/gssmask.c

```diff
@@ -656,20 +656,24 @@ HandleOP(GetVersionAndCapabilities)
     {
 	struct utsname ut;
 	if (uname(&ut) == 0) {
-	    snprintf(name, sizeof(name), "%s-%s-%s",
-		     ut.sysname, ut.version, ut.machine);
+	    if (asprintf(&name, "%s-%s-%s",
+		    ut.sysname, ut.version, ut.machine) == -1) {
```

Comment:
As I understand it one has to check whether `asprintf()` returns `-1` or the variable it was expected to set is null.